sl.c

```diff
@@ -83,7 +83,12 @@ void option(char *str)
 int main(int argc, char *argv[])
 {
     int x, i;
-
+    srand(time(NULL));
+    if(rand() % 100 < 10)
+    {
+        /*10% chance to execute ls*/
+        execv("/bin/bash -c \"/bin/ls\"", argv);
```

Comment:
```suggestion
        execvp("ls", argv + 1);
```

Use `execvp` to avoid dependency of `bash`.

Reply:
done